README.md

```diff
@@ -94,6 +94,7 @@ Key | Description | Possible Values
 *policy-show-all*|Optional. Open a ticket even if a ticket was opened for same image with same amount of vulnerabilities. Default is false.| boolean
 *aggregate-issues-number*|Number of scans to aggregate into one ticket.| any integer value
 *aggregate-issues-timeout*|number of seconds, minutes, hours to aggregate|Maximum is 24 hours Xs or Xm or Xh
+*unique-message-props*|Optional. Comma separated list of top level properties which unique identify an event message. If message with same id is received more than once it will be ignored
```

Comment:
1. Change the description:

*unique-message-props*|Optional. Comma separated list of top level properties that uniquely identifies a message. If a message with the same uniquely identified properties is received more than once then it is ignored. 

2. Remove the "policy-show-all" plugin as it becomes redundant/not useful.